bin/list_schemes.sh

```diff
@@ -2,7 +2,17 @@
 
 set -o pipefail
 
-xcrun xcodebuild -list "$@" 2>/dev/null \
-  | awk '/Schemes:/,0' \
-  | tail -n +2 \
-  | sed -e "s/^[[:space:]]*//"
+ignore_pattern="$1"
+
+if [[ "$ignore_pattern" == "-workspace" ]]; then
+    xcrun xcodebuild -list 2>/dev/null \
```

Comment:
I think these are losing the reference to the build target now, right? We need to keep passing this through to make sure we're listing the schemes in the selected project/workspace, as opposed to the first one `xcodebuild` finds.